packages/authentication/README.md

```diff
@@ -334,10 +340,23 @@ const useEmulator = async () => {
 * [`getCurrentUser()`](#getcurrentuser)
 * [`getIdToken(...)`](#getidtoken)
 * [`isSignInWithEmailLink(...)`](#issigninwithemaillink)
+* [`linkWithApple()`](#linkwithapple)
+* [`linkWithEmailAndPassword(...)`](#linkwithemailandpassword)
+* [`linkWithEmailLink(...)`](#linkwithemaillink)
+* [`linkWithFacebook()`](#linkwithfacebook)
+* [`linkWithGameCenter()`](#linkwithgamecenter)
```

Comment:
`GameCenter` is currently not supported by this plugin. We can add support in another PR.